### ISSUE TITLE
update-meta-files.sh failed

### ISSUE THREAD

**Issue by Cinux90** [while-true-do/ansible-galaxy-skeleton#6]
### Type of Report

Bug

### Actual behaviour
update-meta-file.sh -t and update-meta-file.sh -m failed as following:
```
cinux@cinux-t420 ~/wtd/workspace/ansible-galaxy-skeleton (master) $ ./update-meta-files.sh -m
Downloading Meta Files
----------------------
Downloading https://raw.githubusercontent.com/while-true-do/ansible-galaxy-skeleton/master/
./update-meta-files.sh: Zeile 72: /home/cinux/wtd/workspace/ansible-galaxy-skeleton/https://raw.githubusercontent.com/while-true-do/ansible-galaxy-skeleton/master/: Datei oder Verzeichnis nicht gefunden
cinux@cinux-t420 ~/wtd/workspace/ansible-galaxy-skeleton (master) $ ./update-meta-files.sh -t
Downloading Test Files
----------------------
Downloading https://raw.githubusercontent.com/while-true-do/ansible-galaxy-skeleton/master/tests/
./update-meta-files.sh: Zeile 62: /home/cinux/wtd/workspace/ansible-galaxy-skeleton/tests/https://raw.githubusercontent.com/while-true-do/ansible-galaxy-skeleton/master/tests/: Datei oder Verzeichnis nicht gefunden
```
I'm not sure if its only by me or in gerneral, but on an centos7 machine its also.
As for now i have not find any solution. The script itself looks fine, expect the ongoing changes in pullrequest #5 

### Expected behaviour

To Download the needed files.

### Specifications

Bash Version 4.4.12 / 4.2.46
OS: Manjaro, centos7

**Comment by daniel-wtd** [comment]
the update script in the master does not work, yet. After #5 this should work.

Since this is pulling from the skeleton itself, we have to recheck after #5 was merged.

**Comment by daniel-wtd** [comment]
found the issue will be fixed in #5

**Comment by Cinux90** [comment]
checked with #5 and its will fix the issue. Dont know how i can overlooked it.
Thanks :)
